src/App.tsx

```diff
@@ -1,25 +1,20 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
-import "./styles/tailwind.output.css";
+import React from 'react';
+import './App.scss';
+import LoginPage from './pages/login/LoginPage';
+import SignupPage from './pages/signup/SignupPage';
+import './styles/tailwind.output.css';
+import { Switch, Route } from 'react-router-dom';
+import ErrorPage from './pages/error/ErrorPage';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
+      {/* <LoginPage /> */}
```

Comment:
Remove the comment here when it is unused

Reply:
Fixed!